doc/how_to/components/custom_reactiveHTML.md

```diff
@@ -0,0 +1,180 @@
+# Build Components from Scratch
```

Comment:
Would suggest to move these to a "Building Custom Components" section in the advanced section of the How To.

Reply:
hmm it makes sense to me to have subsections by topic OR difficulty, but not both; i.e. to not have an 'Advanced' section of how-to's. I could imagine that each topic section could have advanced how-to's and we probably wouldn't want to separate them from related guides (of any difficulty) in the main view. I think I prefer some sort of minimal tag like I've done here to label the advanced ones as a sort of warning. I don't think anyone will start their search looking specifically for 'Advanced' guides of any topic... 

I'm ok with moving them both to a new section about extending Panel

Reply:
We already have an advanced section. It's simply a heading in the index.md.

Reply:
I appreciate what you're saying though, thematic subsections should be grouped together whether they are simple or advanced. I think it's reasonable to break down the actual subsections into core and advanced groupings.